flycheck-haskell.el

```diff
@@ -251,13 +251,21 @@ buffer."
   "Process the a Cabal CONFIG."
   (let-alist config
     (setq-local flycheck-ghc-search-path
-                (append .build-directories .source-directories
+                (append (if (string-equal (car flycheck-haskell-runghc-command)
+                                          "stack")
```

Comment:
I was under the impression that this command that's used to run actual checker, so if we run it with `stack` we obviously should give it directories Stack works with. Sorry, how can I properly detect what is used to run actual checker?